### ISSUE TITLE
Landmarks accessibility support for visual keyboard users in 2.4.1 ByPass Blocks

### ISSUE THREAD

**Issue by kimviens** [w3c/wcag#2533]
This issue is similar to #1712. 

In the WCAG sufficient techniques, it is said that [implementing landmarks would pass ByPass Blocks](https://www.w3.org/WAI/WCAG22/Techniques/aria/ARIA11) as long as it "groups blocks of repeated material in a way that can be skipped".

The problem is that, for visual keyboard-only users, the knowledge of the existence of landmarks is very little. Most keyboard-only users I know do not navigate using landmarks.

There is a great extension created by [Matatk](https://github.com/matatk) called [Landmarks](http://matatk.agrip.org.uk/landmarks/). But not a lot of people uses it, or it is not one of the main tools that visual keyboard users use when it should be. 

My question is: Is it our responsibility to increase the awareness about the extension to increase it's adoption? Would that improve the accessibility support?

I am willing to work to spread awareness, I just want to know if I should.

**Comment by alastc** [comment]
Hi @kimviens,

> Is it our responsibility to increase the awareness about the extension to increase it's adoption? Would that improve the accessibility support?

I'm not sure who 'us' is in this context, but I'd say there is a general responsibility on people who know how to improve a situation.

Personally, I don't have the contacts or awareness of where people who would benefit congregate. (We do usability testing at work, but that's not a lot of people per year that would benefit from this specific tool.)

If you are involved with, or have contacts at a charity that supports people with motor impairments, that might be a good starting point?

**Comment by kimviens** [comment]
@alastc Thank you for answering my inquiry. Where I work I have contacts that can move things forward with the adoption of the extension, I wasn't sure if I should. 

You said: "I'd say there is a general responsibility on people who know how to improve a situation" and I think it is a fair point. I believe that I will start to move things forward on my side. Thank you!
